### PR TITLE
Features/operators-guide

### PR DESCRIPTION
**Pull request by serinko**
# Description

Staring a new mdbook with edited node setup manuals and additional faq for Nym platform operators

### CODE REVIEW

File: documentation/operators/src/introduction.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+If you are new to Nym and want to learn about the mixnet, how to integrate with the network, developer tutorials and quickstart guides, check out the [Developer Portal](https://nymtech.net/developers/).
+
+If you want to go deeper into Nym architecture and various software such as different, application clients, existing applications like the desktop wallet, mixnet explorer or SDKs, visit [Nym's technical documentation](https://nymtech.net/docs/).
+
+If you are looking for information on grants and beta-release Nym apps, check out the [Shipyard](https://shipyard.nymtech.net) site.
+
+
+## Popular pages
+
+<!---serinko: not sure how much to include from the docs and developers books in here--->
```
Comment (mfahampshire) [inline documentation/operators/src/introduction.md:14]: I think for the moment we do: 

* node setup guides 
* node maintenance and troubleshooting 

I'm not sure we need client setup... maybe keep that in the docs

File: documentation/operators/src/nodes/mix-node-setup.md [hunk 2]
```diff
@@ -0,0 +1,700 @@
...
+
+For now, see the below rough specs:
+
+- Processors: 2 cores are fine. Get the fastest CPUs you can afford.
+- RAM: Memory requirements are very low - typically a mix node may use only a few hundred MB of RAM.
+- Disks: The mixnodes require no disk space beyond a few bytes for the configuration files.
+
+## Mix node setup
+
+Now that you have built the codebase, set up your wallet, and have a VPS with the `nym-mixnode` binary, you can set up your mix node with the instructions below.
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/mix-node-setup.md:66]: Missing a link or note to build the repo first here - remember we also have the binaries in the github releases page

**Comment by mfahampshire** [comment]
Overall comment: I think we change the `bonding via wallet (recommended)` and `bonding via CLI (power users)` to just `bonding via wallet (GUI)` and `bonding via CLI` or something like that?

File: documentation/operators/src/faq.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<!---
```
Comment (mfahampshire) [inline documentation/operators/src/faq.md:1]: You can use a load of the stuff from here too https://nymtech.net/docs/nodes/troubleshooting.html

Comment (serinko) [reply documentation/operators/src/faq.md:1]: This I am expecting to use as well. And separate it to troubleshooting and to FAQ, where also some basics on the active set etc shall be included.

**Pull request closed without merging**